ClangPowerTools/ClangPowerTools/Options/ViewModel/ClangTidyOptionsView.cs

```diff
@@ -195,12 +209,26 @@ public override void LoadSettingsFromStorage()
 
       SetEnvironmentVariableTidyPath();
     }
-
-
     #endregion
 
 
     #region Private Methods
+    private bool DoesSolutionDirectoryContainsClangTidyFile()
+    {
+
+      VsServiceProvider.TryGetService(typeof(DTE), out object dte);
+      var solution = (dte as DTE2).Solution;
+
+      if (solution == null || !solution.IsOpen)
```

Comment:
Use boolean comparison instead of `!` operator for better code readability.

Reply:
Ok, done!